packages/soporteremoto/Makefile

```diff
@@ -1,39 +1,13 @@
-#
-# Copyright (C) 2006-2013 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v3.
-#
+LIME_TITLE:=soporteremoto allows AlterMundi to provide remote support through SSH root access
+LIME_MAINTAINER:=Gui Iribarren <gui@altermundi.net>
+LIME_DEPENDS:= +zerotier +auto-usb-wwan +kmod-ath9k-htc +ath9k-htc-firmware
 
-include $(TOPDIR)/rules.mk
-
-PKG_NAME:=soporteremoto
-PKG_VERSION=$(GIT_COMMIT_DATE)-$(GIT_COMMIT_TSTAMP)
-GIT_COMMIT_DATE:=$(shell git log -n 1 --pretty=%ad --date=short . )
-GIT_COMMIT_TSTAMP:=$(shell git log -n 1 --pretty=%at . )
-
-include $(INCLUDE_DIR)/package.mk
-
-define Package/$(PKG_NAME)
-  SECTION:=utils
-  CATEGORY:=Utilities
-  TITLE:=$(PKG_NAME) allows AlterMundi to provide remote support through SSH root access
-  MAINTAINER:=Gui Iribarren <gui@altermundi.net>
-  DEPENDS:= +zerotier +auto-usb-wwan +kmod-ath9k-htc +ath9k-htc-firmware
-endef
-
-define Package/$(PKG_NAME)/description
+define LIME_DESCRIPTION
   Installs auto-usb-wwan to provide a zero-configuration WWAN via USB dongle
-  zerotier to provide VPN transport
-  and installs a predefined set of ssh authorized_keys of AlterMundi members
-  that will be granted root access
-endef
-
-define Build/Compile
+  zerotier to provide VPN transport and installs a predefined set of ssh
+  authorized_keys of AlterMundi members that will be granted root access.
 endef
 
-define Package/$(PKG_NAME)/install
-	$(INSTALL_DIR) $(1)/
-	$(CP) ./files/* $(1)/
-endef
+include ../../libremesh.mk
 
-$(eval $(call BuildPackage,$(PKG_NAME)))
+# call BuildPackage - OpenWrt buildroot signature
```

Comment:
I have a semi-off topic question. If I want to override the `install` definition, what is the way to do it?

Addin the install definition after the include of libremesh.mk and Calling `$(eval $(call BuildPackage,$(PKG_NAME)))` again here?

Reply:
I think after the inclusion similar to [here](https://github.com/openwrt/luci/blob/7edd635026594da577b73f059c0d8b95d653f8fc/modules/luci-base/Makefile). However can't you improve the `libremesh.mk` version?

Reply:
Thanks, yes maybe I can improve the LibreMesh.mk version (after understanding how does it work), I was just asking not implying it can't be done. I don't know if it is only me or makefiles are very difficult to understand, follow and debug so I try to maintain them as tiny and simple as possible.

Reply:
I also find Makefiles an annoying and complex overhead, that's why I love the approach to have a centralised one in libremesh.mk :)